tutorials/3d/physical_light_and_camera_units.rst

```diff
@@ -0,0 +1,260 @@
+.. _doc_physical_light_and_camera_units:
+
+Physical light and camera units
+===============================
+
+Why use physical light and camera units?
+----------------------------------------
+
+Since video game graphics generally don't aim for realism, Godot uses arbitrary
+units by default to favor usability. This applies to light color and energy,
+camera field of view, exposure, and more.
+
+Advantages
+^^^^^^^^^^
+
+If you aim for photorealism in your project, using real world units as a basis
+can help make things easier to adjust. References for real world materials,
+lights and scene brightness are wildly available online. This can also be useful
+when porting a scene from other 3D software that uses physical light units (such
+as Blender).
+
+Disadvantages
+^^^^^^^^^^^^^
+
+Using physical light and camera units will not automatically make your project
+look *better*. Sometimes, moving away from realism can actually make a scene
+look better to the human eye. Also, using physical units also requires a greater
+amount of rigor compared to non-physical units. Most benefits of physical units
+can only be obtained if the units are correctly set to match real world
+reference.
+
+.. note::
+
+    Physical light units are only available in 3D rendering, not 2D.
+
+Setting up physical light units
+-------------------------------
+
+Physical light units can be enabled separately from physical camera units.
+
+To enable physical light units correctly, there are 4 steps required:
+
+1. Enable the project setting.
+2. Configure the camera.
+3. Configure the environment.
+4. Configure Light3D nodes.
+
+Since physical light and camera units only require a handful of calculations to
+handle unit conversion, enabling them doesn't have any noticeable performance
+impact.
+
+Enable the project setting
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open the Project Settings, enable the **Advanced** toggle then enable
+**Rendering > Lights And Shadows > Use Physical Light Units**. Restart the editor.
+
+Configure the camera
+^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+
+    When physical light units are enabled, you **must** have a
+    :ref:`class_CameraAttributes` resource assigned to the
+    WorldEnvironment node. Otherwise, the 3D editor viewport will
+    appear extremely bright if you have a visible DirectionalLight3D node.
```

Comment:
```suggestion
    When physical light units are enabled and if you have a
    WorldEnvironment node in your scene (i.e. the editor
    Environment is disabled), you **must** have a
    :ref:`class_CameraAttributes` resource assigned to the
    WorldEnvironment node. Otherwise, the 3D editor viewport will
    appear extremely bright if you have a visible DirectionalLight3D node.
```

The editor environment is tuned to work with the editor light, so if they have both present it will look fine.